.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: pre-release-binaries
+on:
+    push:
+      tags:
+        - "v*.*.*"
+
+jobs:
+  build:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    strategy:
+      matrix:
+        goos: [linux, darwin]
+        goarch: [amd64, arm64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - name: Build
+        run: |
+          go run gotest.tools/gotestsum@latest --format github-actions ./...
+          mkdir -p dist
+          GOOS=${{ matrix.goos }} GOARCH=${{ matrix.goarch }} go build -o dist/cofidectl-${{ matrix.goos }}-${{ matrix.goarch }} ./cmd/cofidectl/main.go
```

Comment:
We did briefly discuss using https://goreleaser.com/. Have you ever tried it?

Reply:
Yeah, though ended up going with: https://github.com/release-drafter/release-drafter

Maybe we should try that? Then we just click "release" on github and it'll create a tag with a reasonable version number and publish a release with release notes. It'll also use all our commit prefixes etc...